gradle.properties

```diff
@@ -1,5 +1,6 @@
 org.gradle.jvmargs=-Xmx4g -XX:MaxMetaspaceSize=1536m -XX:MaxPermSize=1g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8 -XX:+UseParallelGC
 org.gradle.parallel=true
+org.gradle.caching=true
```

Comment:
This should not be set since we want a clean build on CI

Reply:
alright, maybe we should just add `--no-build-cache` in the gradle props for ci?